modules/spdy/src/test/java/org/glassfish/grizzly/spdy/NIOInputSourcesTest.java

```diff
@@ -503,110 +503,6 @@ public void doWrite(FilterChainContext ctx) throws IOException {
 
     }
 
-    /**
-     * Test ReadHandler.onError to be notified, when client unexpectedly
-     * terminates the connection
-     */
-    @SuppressWarnings({"unchecked"})
-    @Test
-    public void testDisconnect() throws Throwable {
```

Comment:
I think I'm not seeing the reason why we moved this test to its own TestCase.
Was this just a refactor or there was a problem otherwise?

Reply:
It failed when running with the same "statics" as the other tests. I extracted it then